examples/simple_system/ibex_simple_system.cc

```diff
@@ -26,8 +24,13 @@ int main(int argc, char **argv) {
             << "==================" << std::endl
             << std::endl;
 
-  retcode = simctrl.Exec(argc, argv);
+  if(simctrl.Exec(argc, argv)) {
+    return 1;
+  }
 
+  // TODO: Exec can return with "true" (e.g. with `-h`), but that does not mean
+  // `RunSimulation()` was executed. The folllowing values will not be useful
+  // in this case.
```

Comment:
That's good enough for now, and I'll follow with an OnFinish() event handler, where this code should go. (This is something that Tom will need in #484 as well.)